myhpom/auth.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+class EmailAuthBackend(ModelBackend):
+    """
+    Authenticate a user via email (which is unique in this system) rather than username.
+    """
+    def authenticate(self, username=None, password=None, **kwargs):
+        User = get_user_model()
+        try:
+            user = User.objects.get(email=username)
+        except User.DoesNotExist:
+            return None
+        else:
+            if user.check_password(password):
```

Comment:
Add `and self.user_can_authenticate(user)` is included to prevent inactive users from logging in.